quick-start-aks.md

```diff
@@ -0,0 +1,198 @@
+# Quick start
+
+In this guide we will see how to start the [Initium Platform](https://github.com/nearform/initium-platform) on a AKS cluster and deploy an application to it from a GitHub action using the [Initium CLI](https://github.com/nearform/initium-cli).
+
+## Install Azure CLI Locally
+Ignore this step if you already have Azure CLI setup.
+
+Install `azure-cli` for your OS by following the official guide [Install az cli official](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-macos). 
+
+## Login to Azure
+
+Run below command. Note that there are different way we can authenticate with Azure using Azure CLI refer [MS Docs Az login](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli)
+
+In this quick start guide we use the option `Sign in interactively` which would open a browser window to get the login credentials.
+
+``` bash
+az login
+```
+
+In case you have access to multiple Azure subscription set the account from CLI as below
+
+``` bash
+az account -s <Subscription name or Id>
+```
+## Create AKS Cluster for Dev/Test Purpose
+
+From the command line run below command to create a AKS cluster. Note that you need to create a new resource group to host the AKS resource.
+
+``` bash
+AKS_RESOURCE_GROUP="<Your Resource Group Name>"
```

Comment:
Can we move this to a separate code block and I'm not sure this is going to work if the user doesn't export the variables.

Reply:
I tested this and it worked without the export. But anyway added the export now.